dashboards-observability/public/components/common/search/autocomplete.tsx

```diff
@@ -374,6 +386,7 @@ export function Autocomplete({
         React.KeyboardEvent
       >(
         {
+          openOnFocus: true,
```

Comment:
what about `defaultActiveItemId: 0,`? i think most IDE completions will auto select first match

Reply:
I can add this!